custom_components/tuya_local/devices/solar_inverter_gtb_series.yaml

```diff
@@ -0,0 +1,161 @@
+name: Smart Inverter GTB Series
```

Comment:
Top level name should be generic - such as "Solar inverter".
Stay away from redundant words like "smart" in the naming (they can go in the name under products if they are part of the branding)